### ISSUE TITLE
Niemann-Pick disease is not a neoplasm

### ISSUE THREAD

**Issue by paolaroncaglia** [EBISPOT/efo#292]
EFO:1001380 ‘Niemann-Pick disease’ is currently classified as a neoplasm (see https://www.ebi.ac.uk/ols/ontologies/efo/terms?short_form=EFO_1001380), but it’s a metabolic disorder instead (see https://en.wikipedia.org/wiki/Niemann–Pick_disease). No idea where that parentage came from - not from the DOID or the NCIt equivalents, at least as they are now (I haven’t checked ICD10, MSH and MedRA yet).

MONDO’s classification looks better (see http://www.ontobee.org/ontology/MONDO?iri=http://purl.obolibrary.org/obo/MONDO_0001982). We should classify as subclass of sphingolipidosis (as indeed all 4 children of the terms are already) or other metabolic disorder (need to look more closely).

We also need to check and, if necessary, fix axioms for the child EFO Orphanet:646 ‘Niemann-Pick disease type C’ (now subclass of, among others, eye neoplasm, brain neoplasm, and a couple other axioms that may be better represented by different relations). 

Lastly, the definition of EFO:1001380 ‘Niemann-Pick disease’ now reads “_The classic infantile form of Niemann-Pick Disease_, caused by mutation in SPHINGOMYELIN PHOSPHODIESTERASE...” which sounds more like a subclass of the disease than its generic form; check.

**Comment by paolaroncaglia** [comment]
Note for self/EFO editor:

- [ ] Edit EFO:1001380 ‘Niemann-Pick disease’:
Make it subclass of EFO Orphanet:79225 ‘Sphingolipidosis’ instead of ‘neoplasm’;
Classify its 4 children (Niemann-Pick disease type A/B/C/E) under it rather than under EFO Orphanet:79225 ‘Sphingolipidosis’;
Modify definition to read: “A group of inherited, severe metabolic disorders in which sphingomyelin accumulates in lysosomes in cells. The lysosomes normally transport material through and out of the cell.”;
Add definition citations MONDO:0001982, https://en.wikipedia.org/wiki/Niemann–Pick_disease;
Edit synonyms: remove the ones referring to types A/B/C.

- [ ] Edit EFO Orphanet:646 ‘Niemann-Pick disease type C’:
Remove subclasses ‘eye neoplasm’, ‘tauopathy’, ‘brain neoplasm’, ‘supranuclear oculomotor palsy’, ‘Unclassified primitive or secondary maculopathy’, and check https://en.wikipedia.org/wiki/Niemann–Pick_disease,_type_C to see if those disorders should be captured by other axioms instead (has disease feature?).

**Comment by paolaroncaglia** [comment]
The Wikipedia entry for ‘Niemann-Pick disease type C’ (https://en.wikipedia.org/wiki/Niemann–Pick_disease,_type_C) reads:

"_Progressive neurological disease is the hallmark of Niemann–Pick type C disease_, and is responsible for disability and premature death in all cases beyond early childhood. Classically, children with NPC may initially present with delays in reaching normal developmental milestones skills before manifesting cognitive decline (dementia).
Neurological signs and symptoms include: [a list of numerous and diverse specific disorder follows; symptoms also vary as the disease progresses]."

The list of symptoms is such that it would be difficult, and potentially incorrect, to try and capture all of them. We could add a broad axiom along the lines of
‘Niemann-Pick disease type C’ disease_has_feature 'nervous system disease'.
@simonjupp @zoependlington what do you think?

Thanks.

**Comment by paolaroncaglia** [comment]
At our OntoTools scrum today, we resolved that we may not want to capture neurological phenotypes for Niemann-Pick disease type C (see https://github.com/EBISPOT/efo/issues/292#issuecomment-441980453). The current neurological parents are incorrect (it's a metabolic disease, resulting in severe phenotypes across the body including neurological symptoms, but we wouldn't classify it as a nervous system disease) and presumably derive from the Orphanet peculiar structure (http://www.orpha.net/ORDO/Orphanet_646). We also noticed that MONDO has the same issue (http://purl.obolibrary.org/obo/MONDO_0018982), so I'll open a ticket with them to edit the term consistently.
